Dockerfile

```diff
@@ -1,7 +1,16 @@
-FROM golang:alpine3.11
+FROM golang:alpine3.11 as base
 
 WORKDIR /app
 COPY . /app
 
+ENV CGO_ENABLED=0
 RUN go build -o bin/naveed
-CMD ["/app/bin/naveed"]
+
+FROM busybox
+
+COPY --from=base /app/bin/naveed /usr/bin/naveed
+COPY --from=base /app/naveed.ini /
```

Comment:
this works because `/` is `CMD`'s working directory?

Reply:
correct